Python/Video_Processing/GetFrames.py

```diff
@@ -1,14 +1,43 @@
 import cv2
 
-cap = cv2.VideoCapture('ENTER VIDEO NAME HERE')
-count = 0
-while cap.isOpened():
-    ret, frame = cap.read()
-    cv2.imshow('window-name', frame)
-    cv2.imwrite("VIDEO_NAME/frame%d.jpg" % count, frame)
-    count = count + 1
-    if cv2.waitKey(10) & 0xFF == ord('q'):
-        break
-
-cap.release()
-cap.destroyAllWindows()
+
+def get_frames_entire_video(video_name):
+    cap = cv2.VideoCapture(video_name)
+    count = 0
+    while cap.isOpened():
+        ret, frame = cap.read()
+        cv2.imshow('window-name', frame)
+        cv2.imwrite("VIDEO_NAME/frame%d.jpg" % count, frame)
+        count = count + 1
+        if cv2.waitKey(10) & 0xFF == ord('q'):
+            break
+    cap.release()
+    cap.destroyAllWindows()
+
+
+def get_frames_of_all_letters(video_name, input_list):
+    cap = cv2.VideoCapture(video_name)
+
+    frame_count = 1
+    list_count = 0
+    current_letter = input_list[0][0]
+    while cap.isOpened():
+        ret, frame = cap.read()
+        if input_list[list_count][1] <= frame_count <= input_list[list_count][2]:
+            try:
+                cv2.imwrite('test_frame%d_%s.jpg' % (frame_count, current_letter), frame)
```

Comment:
write to directory for the letter we are currently on, change name to match {<letter>\<tranlator>_<c or d>_<frame>.png}

Reply:
So example being 'A/07_1.jpg', A being the letter, 07 being the translator number, and 01 being the frame?

Reply:
A/07_C_001.jpg (where C is the C/D on video descriptor) - this is so that when splitting data for train/test there isn't overlap between any videos next time when manually split